src/Symfony/Component/Notifier/Bridge/Clickatell/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+CHANGELOG
+=========
+
+5.3.0
+-----
+
+ * Added the bridge
```

Comment:
```suggestion
5.3
---

 * Add the bridge
```

Fabien documented how to write `CHANGELOG.md` files here, please keep a look and adjust it accordingly.

https://github.com/symfony/symfony-docs/pull/14830

Thanks 🙏

Reply:
Ok thank you for this information.

But `Add the bridge` is not to short ? Why not write `Add notifier bridge for <provider name>` ?

In the Fabien's example, the description is more detailed I find.

Reply:
You are right, but in this case all of the other bridges used this sentence 😃